skribenten-backend/src/main/resources/application-local.conf

```diff
@@ -5,7 +5,7 @@ skribenten {
     port = ${?PORT}
     services {
         pen {
-            url = "https://pensjon-pen-q2.dev.adeo.no/pen/api/"
+            url = ${?PEN_URL}
```

Comment:
kor hentast den frå?

Reply:
Men da vil dette knekke moglegheita for å køyre skribenten lokalt utanfor docker-compose 😞